test/prototype_transforms_kernel_infos.py

```diff
@@ -2100,3 +2118,39 @@ def sample_inputs_convert_dtype_video():
         ),
     ]
 )
+
+
+def sample_inputs_uniform_temporal_subsample_video():
+    for video_loader in make_video_loaders(sizes=["random"], num_frames=[4]):
+        for temporal_dim in [-4, len(video_loader.shape) - 4]:
+            yield ArgsKwargs(video_loader, num_samples=2, temporal_dim=temporal_dim)
```

Comment:
As shown above, the inputs defined here should cover all valid code paths but should otherwise not be excessive. Since there no branches in the whole kernel, the definition is trivial. I simply used `make_video_loaders` and restricted the videos to random spatial sizes and a fixed `num_frames`. On top of that I also added positive `temporal_dim`'s to the mix.

In total we are looking at 

https://github.com/pytorch/vision/blob/9c112935abe400222cca8f9fbc2d8386e0f25e80/test/prototype_common_utils.py#L582-L592

1 `sizes`, 2 `color_space`, 1 `num_frames`, 6 `extra_dims`

https://github.com/pytorch/vision/blob/9c112935abe400222cca8f9fbc2d8386e0f25e80/test/prototype_common_utils.py#L216-L219

1 `dtype`, and 2 `temporal_dim`'s or 24 total sample inputs. That is totally fine and even on the lower end of sample inputs compared to more complicated kernels.

If you are wondering why we are using the `make_video_loaders` here instead of `make_videos`, the reason is rather simple. These sample inputs will be loaded at test collection time and are alive for the whole duration of running the test suite. With this PR, we instantiate ~55k tests. We don't want to instantiate that many tensors. Thus, we are using the `TensorLoader` concept that knows how to instantiate the tensor and already has some metadata about it. For example, you can see that I can access `video_loader.shape` during the sample generation, but the actual values are unknown at this point. The tensor will only be instantiated inside the test with this call

https://github.com/pytorch/vision/blob/9c112935abe400222cca8f9fbc2d8386e0f25e80/test/test_prototype_transforms_functional.py#L96